boards/common/sam0/Makefile.features

```diff
@@ -0,0 +1,2 @@
+# Put other features for this board (in alphabetical order)
+FEATURES_PROVIDED += riotboot
```

Comment:
Can a board with e.g. the Arduino bootloader still have `riotboot` support? I don't think so.

Reply:
Ok so this is only about the `-xpro` boards, not all `sam0` boards - better reflect that in the name.

Reply:
> Can a board with e.g. the Arduino bootloader still have riotboot support?

It's technically doable. One just has to also take into account the bootloader offset.

Reply:
I dropped this change from this PR and moved it to #13864. So now the common sam0 board configuration only contains the openocd configuration. For the moment, I can't a better place for putting it.